proxyd/cache.go

```diff
@@ -176,14 +194,24 @@ func newRPCCache(cache Cache) RPCCache {
 		},
 	}
 	handlers := map[string]RPCMethodHandler{
-		"eth_chainId":                           staticHandler,
-		"net_version":                           staticHandler,
-		"eth_getBlockTransactionCountByHash":    staticHandler,
-		"eth_getUncleCountByBlockHash":          staticHandler,
-		"eth_getBlockByHash":                    staticHandler,
-		"eth_getTransactionByBlockHashAndIndex": staticHandler,
-		"eth_getUncleByBlockHashAndIndex":       staticHandler,
-		"debug_getRawReceipts":                  debugGetRawReceiptsHandler,
+		"eth_chainId":                             staticHandler,
+		"net_version":                             staticHandler,
+		"eth_getBlockTransactionCountByHash":      staticHandler,
+		"eth_getUncleCountByBlockHash":            staticHandler,
+		"eth_getBlockByHash":                      staticHandler,
+		"eth_getTransactionByBlockHashAndIndex":   staticHandler,
+		"eth_getUncleByBlockHashAndIndex":         staticHandler,
+		"debug_getRawReceipts":                    debugGetRawReceiptsHandler,
+		"eth_getBlockByNumber":                    shortLivedHandler,
+		"eth_blockNumber":                         shortLivedHandler,
+		"eth_getBalance":                          shortLivedHandler,
+		"eth_getStorageAt":                        shortLivedHandler,
+		"eth_getTransactionCount":                 shortLivedHandler,
+		"eth_getBlockTransactionCountByNumber":    shortLivedHandler,
+		"eth_getUncleCountByBlockNumber":          shortLivedHandler,
+		"eth_getCode":                             shortLivedHandler,
+		"eth_getTransactionByBlockNumberAndIndex": shortLivedHandler,
+		"eth_getUncleByBlockNumberAndIndex":       shortLivedHandler,
```

Comment:
Should all of these be low TTL? ex. eth_getCode, eth_getBlockByNumber or eth_getBlockTransactionCountByNumber? 

Just curious how these RPCs were chosen, any related data? I get that tip relevant RPCs and rapidly changing data fit this behavior

Reply:
Follow up could be making this configurable but I think that maybe out of scope for the inital PR